wavefront/data_source_alert.go

```diff
@@ -244,7 +305,29 @@ func flattenHostLabelPairs(pairs []wavefront.SourceLabelPair) interface{} {
 
 func flattenHostLabelPair(pair wavefront.SourceLabelPair) map[string]interface{} {
 	tfMap := make(map[string]interface{})
-	tfMap["firing"] = pair.Firing
-	tfMap["host"] = pair.Host
+	tfMap[firingKey] = pair.Firing
+	tfMap[hostKey] = pair.Host
```

Comment:
I started the process of replacing these hard coded strings with constants, but there are a lot and I wanted to get this fix in and not be held up. So I will continue replacing them after this is merged.